addons/website/tests/test_ui.py

```diff
@@ -216,3 +216,8 @@ def test_11_website_snippet_background_edition(self):
             'mimetype': 'image/png',
         })
         self.start_tour('/', 'snippet_background_edition', login='admin')
+
+    def test_12_edit_translated_page_redirect(self):
+        self.env['base.language.install'].create({'lang': 'nl_NL'}).lang_install()
+        self.start_tour("/nl_NL", 'edit_translated_page_redirect', login='admin', timeout=120)
```

Comment:
```suggestion
        self.start_tour("/nl", 'edit_translated_page_redirect', login='admin', timeout=120)
```
Going to `/nl_NL` is performing a redirect to `/nl` for no reason.
nl_NL -> Locale Code
nl -> URL Code